### DISCUSSION TITLE
Deno Client Tests Demo API

### DISCUSSION THREAD

**Discussion by mathe42** [gotenberg/gotenberg#414] [Q&A] [answered]
Hi,
I currently work on a client for Deno (see https://deno.land/x/gotenberg). Most API is finished. (I need to write some helpers)

As I'm currently working on the Tests is it okay to use the DEMO-API at https://demo.gotenberg.dev/ for that?

**Comment by gulien** [comment, marked as answer]
~~Please do~~* 😄 And keep me in the loop so that I can add your client on the [awesome list](https://github.com/gotenberg/awesome-gotenberg) and the documentation!

EDIT

*Actually the demo API will evolve so it might not be the best use case. For https://github.com/gotenberg/gotenberg-php, I never call the API in the tests, but make sure the requests are well-formed.

**Reply by mathe42** [reply]
Just released version 0.4.0. I think API is stable now. I will use it in my next project wich is deno based so when that all works out I will release 1.0 - feel free to add it to the awasome list or wait ;D

**Reply by gulien** [reply]
@mathe42 Will do! Thanks for your work 😄
